### PR TITLE
7721, 7722, 7726 updates to outputs

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:*
    bcgov/entity#7721
    bcgov/entity#7722
    bcgov/entity#7726

*Description of changes:*
 - Update Incorporation Application output
 - Updated Notice of Articles output
 - Update Alteration Notice output

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/report-templates/template-parts/common/statement.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  </div>
+{% elif business.legalType == 'ULC' %}
+  <div class="no-page-break">
+    <div class="section-title mt-4">B.C. Unlimited Liability Company Statement</div>
+    <div class="section-data mt-4">
+      The shareholders of this company are jointly and severally liable to satisfy the debts and
+      liabilities of this company to the extent provided in section 51.3 of the Business Corporations Act.
+    </div>
+  </div>
+{% endif %}
```
Comment (severinbeauvais) [inline legal-api/report-templates/template-parts/common/statement.html:26]: Is there a case for LTD also?

File: legal-api/report-templates/template-parts/common/statement.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{% if business.legalType == 'BEN' %}
+  <div class="no-page-break">
+    <div class="section-title mt-4">B.C. Benefit Company Statement</div>
```
Comment (severinbeauvais) [inline legal-api/report-templates/template-parts/common/statement.html:3]: Are we going with "BC" or "B.C." ? I think it's inconsistent atm.

Comment (vysakh-menon-aot) [reply legal-api/report-templates/template-parts/common/statement.html:3]: Yeah Linda asked me to make this change yesterday.

File: legal-api/report-templates/template-parts/common/statement.html (hunk 1 above)
Comment (vysakh-menon-aot) [reply legal-api/report-templates/template-parts/common/statement.html:26]: For now these are the statements available. We can extend it when we have more

**Review by severinbeauvais — APPROVED** [review]
LGTM but please wait for 1 more approval.

**Review by andrepestana-aot — APPROVED** [review]
👍

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**